src/app/search/search-tool-table/search-tool-table.component.html

```diff
@@ -57,6 +53,12 @@
         </a>
       </mat-cell>
     </ng-container>
+    <ng-container matColumnDef="starredUsers">
+      <mat-header-cell *matHeaderCellDef mat-sort-header>Stars</mat-header-cell>
+      <mat-cell class="description-cell" *matCellDef="let tool">{{ !tool?.starredUsers || tool?.starredUsers.length === 0 ? '' : tool?.starredUsers.length }}
+        <mat-icon class = "star" *ngIf="tool?.starredUsers?.length > 0 " >star_border</mat-icon>
```

Comment:
Although your HTML is syntactically valid, I don't think there are spaces around the equals sign of attributes anywhere else, so I would remove them (`class="star"` instead of `class = "star")`